WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -78,13 +78,11 @@ extension ReaderTabView {
     }
 
     private func populateTabBar(with items: [ReaderTabItem]) {
-        UIView.animate(withDuration: Appearance.tabBarAnimationsDuration) {
-            self.tabBar.items = items
-            guard let tabItem = self.tabBar.items[self.tabBar.selectedIndex] as? ReaderTabItem else {
-                return
-            }
-            self.buttonsStackView.isHidden = tabItem.shouldHideButtonsView
+        self.tabBar.items = items
```

Comment:
One last nitpick – we could remove the `self` references from here now it's no longer in a block :)